plugin-server/src/main/ingestion-queues/session-recording-v2/teams/team-service.ts

```diff
@@ -32,4 +32,29 @@ export class TeamService {
             consoleLogIngestionEnabled: teamConfig.consoleLogIngestionEnabled,
         }
     }
+
+    private async fetchTeamTokensWithRecordings(): Promise<Record<string, TeamIDWithConfig>> {
+        return fetchTeamTokensWithRecordings(this.postgres)
+    }
+}
+
+export async function fetchTeamTokensWithRecordings(client: PostgresRouter): Promise<Record<string, TeamIDWithConfig>> {
```

Comment:
question: Any reason why we're keeping this and not switching to the lazy loader?

Reply:
Ok, thanks, was just curious – happy to move forward as it is.